.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,10 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
I suppose this file was not committed intentionally, right?

Reply:
Correct, my IDE must have added it automatically. Didn't notice it was there. File can be removed.